cqc/MessageHandler.py

```diff
@@ -296,41 +336,10 @@ def _process_command(self, cqc_header, length, data, is_locked=False):
                 msg = self.create_return_message(cqc_header.app_id, CQC_ERR_GENERAL, cqc_version=cqc_header.version)
                 self.return_messages.append(msg)
                 return False, 0
+            
             if succ is False:  # only if it explicitly is false, if succ is None then we assume it went fine
                 return False, 0
 
-            # Check if there are additional commands to execute afterwards
```

Comment:
This removed part was the backend implementation of the CQC Sequence. I removed the CQC Sequence both from the frontend as well as from the backend because it is 1) undocumented, and 2) has no added functionality (at least not in the current implementation). I verified with Leon, and he confirmed that sequences have no added functionality.

Reply:
@VictorPrins I think there has been some miss-communication here! The header `CQCSequenceHeader` is indeed unnecessary since the sequence can anyway be recognized by the `action`-flag. However, the use of a sequence (by setting `action=True` is still a useful feature and should not be removed. @LeonWubben mentioned that you had trouble implementing the logic while keeping the sequence functionality. Could you elaborate on that?

Reply:
As far as I know, the only functionality of the `action` flag is that it lets the backend process command headers recursively. This is also visible in the lines below: on line 311 of the removed code, the method is recursively invoked. This adds unnecessary complexity for no good reason, in my opinion. Without the action flag, it is still possible to send multiple commands at once. For instance, consider the following CQC program (in pseudo-CQC):

```
CQC Header. Type = CQCType.COMMAND. Length = 3 command headers
Command Header. Instruction = X. Action = False
Command Header. Instruction = Y. Action = False
Command Header. Instruction = Z. Action = False
```

Even though `action` is set to `False`, all three commands will be executed sequentially. This is my understanding of sequences and the `action` flag. Am I missing something?

Regarding the trouble with the logic: During the design phase of my project, I consulted Leon to discuss some ideas and possible solutions. At that point, I had some questions regarding sequences. All these questions were answered and there are no problems anymore. In my final implementation, the sequence functionality works perfectly (albeit without the action flag, and without sequence headers).

Reply:
@VictorPrins, ah yes I see the point, one would instead use the length of the CQC header to know the extent of the sequence. I'll open an issue to decide if we should remove/deprecate the `action` flag completely.